### DISCUSSION TITLE
Add git hash to biome --version

### DISCUSSION THREAD

**Discussion by qarmin** [biomejs/biome#4633] [Ideas]
Currently self-compiled biome for `biome --version` shows 0.0.0

Ruff `ruff version` shows `ruff 0.8.0+1377 (ac23c9974 2024-11-24)` which is a lot of more helpful

Code to gather git info is here

https://github.com/astral-sh/ruff/blob/efe54081d668b0a3091e152f833bd7f90f44f49d/crates/ruff/build.rs#L63

**Comment by ematipico** [comment]
I don't doubt it's more helpful, but those numbers don't mean much, other than the date.

Would you mind explaining those numbers?

**Reply by qarmin** [reply]
ruff version+commit_distance_since_start (short_hash, commit_date)

for me the most important is just hash, because I wouldn't have to search to see what commit I was using to compile app
